scripts/missions/mole_insertion.lua

```diff
@@ -681,17 +681,18 @@ local function moleMission( script, sim )
 	script:addHook( DBhack )
 	script:waitFor( FINISHED_DBHACK )
 	hackConsole:destroyTab()
-	sim:getTags().MM_DBhack_finished = true	
+	sim:getTags().MM_DBhack_finished = true		
```

Comment:
lol, you removed trailing whitespace on one location but added it elsewhere, lmao